synapse/res/username_picker/script.js

```diff
@@ -0,0 +1,116 @@
+let inputField = document.getElementById("field-username");
+let inputForm = document.getElementById("form");
+let submitButton = document.getElementById("button-submit");
+let message = document.getElementById("message");
+
+// Remove input field placeholder if the text field is not empty
+let switchClass = function(input) {
```

Comment:
At some point I updated the style of `login.js` to use named functions instead, e.g. `function switchClass` instead of anonymous functions that get stored into a variable. I'm not sure if there's a huge difference 🤷 , would be nice if we had linting / style enforcement for HTML/JavaScript.

Reply:
have switched to named functions